ddtrace/contrib/django/utils.py

```diff
@@ -88,6 +88,16 @@ def get_request_uri(request):
 
     # Build request url from the information available
     # DEV: We are explicitly omitting query strings since they may contain sensitive information
-    return parse.urlunparse(
-        parse.ParseResult(scheme=request.scheme, netloc=host, path=request.path, params="", query="", fragment="",)
-    )
+    urlparts = dict(scheme=request.scheme, netloc=host, path=request.path)
+    urlparts_omit = ["params", "query", "fragment"]
```

Comment:
I wonder if we can optimize this a bit by just having it be a dict so we don't need to iterate and build a dict later for `.update()`

Reply:
Can we use `None` instead of an empty string? would that simplify this part?